lib/core/src/lib/breaking-changes.md

```diff
@@ -0,0 +1,10 @@
+6.0.0-beta.1
+- CheckAllowableOperationDirective: Moved from ADF Core to ADF content services
```

Comment:
is this a new standard or documenting breaking changes? how this file is going to be discovered after the release, will be linked or something?

Reply:
Yes, we have chosen with @popovicsandras to keep track of all the moved entities related to the beta versions, we have to chose if remove it or not after the release